### PR TITLE
Update docs with the recent changes

### PR DESCRIPTION
**Pull request by chetan-rns**

### CODE REVIEW

File: journey/day1/README.md [hunk 1]
```diff
@@ -34,9 +34,10 @@ $ odo pipelines bootstrap \
   --gitops-repo-url https://github.com/<username>/gitops.git \
   --image-repo quay.io/<username>/<image-repo> \
   --dockercfgjson ~/Downloads/<username>-auth.json \
-  --prefix tst
+  --prefix tst \
+  --output <path to write GitOps resources>
 ```
-In the event of manual installations of sealed secrets as decsribed in the documentation,the flags for service and namespace name will have the following values ```--sealed-secret-ns kube-system and --sealed-secrets-svc sealed-secrets-controller```.
+Executing the above command will generate the GitOps directory and the required resources.
```
Comment (sbose78) [inline journey/day1/README.md:40]: The attached binary needed this, if we are changing this, are awe attaching a new binary?

File: journey/day1/README.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ Day 1 Operations are actions that users take to bootstrap a GitOps system.   Boo
 * [odo pipelines bootstrap](../../commands/bootstrap)
 * [odo pipelines init](../../commands/init)
 
-These commands are similar.  They differ in what gets generated.  The bootstrap command generated a functional GitOps setup including your first application.  The init command only generates CI/CD pipelines and associated resources.  This document describes how to bootstrap GitOps to deliver your first application.
+These commands are similar.  They differ in what gets generated.  The bootstrap command generates a functional GitOps setup including your first application.  The init command only generates CI/CD pipelines and associated resources.  This document describes how to bootstrap GitOps to deliver your first application.
 
 ## Prerequisites
 
-**NOTE**: `odo pipelines` commands are hidden in `Expermential Mode`.  To enable `Expermential Mode` available, please set th `EXPERIEMENTAL` environment variable in the running terminal.
+**NOTE**: `odo pipelines` commands are hidden in `Expermential Mode`.  To enable `Expermential Mode` available, please set the `EXPERIEMENTAL` environment variable in the running terminal.
```
Comment (bigkevmcd) [inline journey/day1/README.md:12]:
```suggestion
**NOTE**: `odo pipelines` commands are experimental, please see [the odo documentation](https://docs.openshift.com/container-platform/4.5/cli_reference/developer_cli_odo/odo-release-notes.html#odo-technology-preview_odo-release-notes) for how to enable experimental mode.
```

File: README.md [hunk 3]
```diff
@@ -52,13 +52,13 @@ A company called *Pet Clinic Supplies Inc* has a big vision but limited investme
...
+They now need a quick fix, on a quick google search they decide to incorporate GitOps, as a [GitOps Day 1 Operations](journey/day1) they want to quickly get started to get their teams up and running . This is when they use our GitOps tools odo bootstrap command to quickly set up their dev, stage and prod environment along with the config repos that create a pipeline for seamless CI/CD with the bootstrapped environments incorporating the [Pipelines Model (aka manifest model)](model). Now the management is happy as they can keep track of the different teams within the organization that are making changes to  their code base and can hold the teams accountable for their actions.
```
Comment (bigkevmcd) [inline README.md:55]:
```suggestion
They now need a quick fix, on a quick google search they decide to incorporate GitOps, as a [GitOps Day 1 Operations](journey/day1) they want to quickly get started to get their teams up and running . This is when they use our GitOps tools odo bootstrap command to quickly set up their dev, stage and prod environment along with the config repos that create a pipeline for seamless CI/CD with the bootstrapped environments incorporating the [Pipelines Model (aka manifest model)](model). Now the management is happy as they can keep track of the different teams within the organization that are making changes to their code base and can hold the teams accountable for their actions.
```

File: README.md [hunk 4]
```diff
@@ -52,13 +52,13 @@ A company called *Pet Clinic Supplies Inc* has a big vision but limited investme
...
 
-Now we have an a team that set out as a start up but went on to be a full fledged organization with a multiple verticals and business set up in different geographical locations leveraging the use of open shift and Gitops with “ Git as the source of truth”. Now's your chance to ramp up your organization using our modernised approach for end to end application delivery. A simple gitops configuration can be seen [here](https://github.com/ishitasequeira/gitops), all achieved with a list of commands listed in [Odo Pipelines Command Reference](commands) , why wait lets get started.
+Now we have an a team that set out as a start up but went on to be a full fledged organization with a multiple verticals and business set up in different geographical locations leveraging the use of open shift and GiOps with “ Git as the source of truth”. Now's your chance to ramp up your organization using our modernised approach for end to end application delivery. A simple gitops configuration can be seen [here](https://github.com/ishitasequeira/gitops), all achieved with a list of commands listed in [Odo Pipelines Command Reference](commands) , why wait lets get started.
```
Comment (bigkevmcd) [inline README.md:61]:
```suggestion
Now we have an a team that set out as a start up but went on to be a full fledged organization with a multiple verticals and business set up in different geographical locations leveraging the use of OpenShift and GitOps with "Git as the source of truth". Now's your chance to ramp up your organization using our modernised approach for end-to-end application delivery. A simple GitOps configuration can be seen [here](https://github.com/rhd-gitops-example/gitops), all achieved with a list of commands listed in [Odo Pipelines Command Reference](commands) , why wait? Lets get started.
```

File: journey/day1/README.md [hunk 5]
```diff
@@ -5,11 +5,11 @@ Day 1 Operations are actions that users take to bootstrap a GitOps system.   Boo
 * [odo pipelines bootstrap](../../commands/bootstrap)
 * [odo pipelines init](../../commands/init)
```
Comment (bigkevmcd) [inline journey/day1/README.md:6]:
```suggestion
```

File: journey/day1/README.md [hunk 6]
```diff
@@ -5,11 +5,11 @@ Day 1 Operations are actions that users take to bootstrap a GitOps system.   Boo
...
 
-These commands are similar.  They differ in what gets generated.  The bootstrap command generated a functional GitOps setup including your first application.  The init command only generates CI/CD pipelines and associated resources.  This document describes how to bootstrap GitOps to deliver your first application.
+These commands are similar.  They differ in what gets generated.  The bootstrap command generates a functional GitOps setup including your first application.  The init command only generates CI/CD pipelines and associated resources.  This document describes how to bootstrap GitOps to deliver your first application.
```
Comment (bigkevmcd) [inline journey/day1/README.md:8]:
```suggestion
The bootstrap command generates a functional GitOps setup including your first application.

This document describes how to bootstrap GitOps to deliver your first application.
```

File: journey/day1/README.md (hunk 1 above)
Comment (DhanashriPalodkar) [reply journey/day1/README.md:40]: yes

Comment (sbose78) [reply journey/day1/README.md:40]: .. but does the current linked binary work without the flags`--sealed-secret-ns kube-system` and `--sealed-secrets-svc sealed-secrets-controller`?

**Pull request merged**